doc/model_details/HillTononi.ipynb

```diff
@@ -0,0 +1,792 @@
+    "- The last term is a re-polarizing current only active during the refractory period, see below. Note that it has a different (faster) time constant than the other currents. It might have been more natural to use the same time constant for all currents and instead adjust $g_{\\text{spike}}$. We follow the original approach here."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Threshold, Spike generation and refractory effects\n",
+    "\n",
+    "The threshold evolves according to [HT05, p 1677]\n",
+    "\n",
+    "\\begin{equation}\n",
+    "\\frac{\\text{d}\\theta}{\\text{d}t} = -\\frac{\\theta-\\theta_{\\text{eq}}}{\\tau_{\\theta}}\n",
+    "\\end{equation}\n",
+    "\n",
+    "The neuron emits a single spike if \n",
+    "- it is not refractory\n",
+    "- membrane potential crosses the threshold, $V\\geq\\theta$\n",
+    "\n",
+    "Upon spike emission,\n",
+    "- $V \\leftarrow E_{\\text{Na}}$\n",
+    "- $\\theta \\leftarrow E_{\\text{Na}}$\n",
+    "- the neuron becomes refractory for time $t_{\\text{spike}}$ (`t_ref` in NEST)\n",
+    "\n",
+    "The repolarizing current is active during, and only during the refractory period:\n",
+    "\\begin{equation}\n",
+    "g_{\\text{spike}} = \\begin{cases}  1  & \\text{neuron is refractory}\\\\\n",
+    " 0 & \\text{else} \\end{cases}\n",
+    "\\end{equation}\n",
+    "\n",
+    "During the refractory period, the neuron cannot fire new spikes, but all state variables evolve freely, nothing is clamped. \n",
+    "\n",
+    "The model of spiking and refractoriness is based on Synthesis model `PulseIntegrateAndFire`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Intrinsic currents\n",
+    "\n",
+    "Note that not all intrinsic currents are active in all populations of the network model presented in [HT05, p1678f].\n",
+    "\n",
+    "Intrinsic currents are based on the Hodgkin-Huxley description, i.e.,\n",
+    "\n",
+    "\\begin{align}\n",
+    "I_X &= g_{\\text{peak}, X} m_X(V, t)^N_X h_X(V, t)(V-E_X) \\\\\n",
+    "\\frac{\\text{d}m_X}{\\text{d}t} &= \\frac{m_X^{\\infty}-m_X}{\\tau_{m,X}(V)}\\\\\n",
+    "\\frac{\\text{d}h_X}{\\text{d}t} &= \\frac{h_X^{\\infty}-h_X}{\\tau_{h,X}(V)}\n",
+    "\\end{align}\n",
+    "\n",
+    "#### Pacemaker current $I_h$\n",
+    "\n",
+    "Synthesis: `IhChannel`\n",
+    "\n",
+    "\\begin{align}\n",
+    "N_h & = 1 \\\\\n",
+    "m_h^{\\infty}(V) &= \\frac{1}{1+\\exp\\left(\\frac{V+75\\text{mV}}{5.5\\text{mV}}\\right)} \\\\\n",
+    "\\tau_{m,h}(V) &= \\frac{1}{\\exp(-14.59-0.086V) + \\exp(-1.87  + 0.0701V)} \\\\\n",
+    "h_h(V, t) &\\equiv 1 \n",
+    "\\end{align}\n",
+    "\n",
+    "Note that subscript $h$ in some cases above marks the $I_h$ channel, \n",
+    "\n",
+    "#### Low-threshold calcium current $I_T$\n",
+    "\n",
+    "Synthesis: `ItChannel`\n",
+    "\n",
+    "##### Equations given in paper \n",
+    "\n",
```

Comment:
Because physicist are a sloppy bunch and call most things with a "=" in the middle an equation. Fixing this here may introduce inconsistencies elsewhere in the notebook, so I prefer to leave it as it is.